DEVLOG.md

```diff
@@ -0,0 +1,24 @@
+## UI todo's
```

Comment:
Can you remove this, we can track this in our [projects ](https://github.com/orgs/ed-pilots-network/projects/4) or you can have it in your local branch if needed.